webrtc.html

```diff
@@ -1243,6 +1243,14 @@
                         "RTCRtpTransceiver">mid</a></code> value to the mid of
                         the corresponding media description.</p>
                       </li>
+                      <li>
+                        <p>If <var>description</var> is of type "answer", then
+                        set <var>transceiver</var>'s <code><a data-for=
+                        "RTCRtpTransceiver">currentDirection</a></code> value
```

Comment:
This is setting a read-only property. See https://github.com/w3c/webrtc-pc/issues/1204.